doc/tutorial/python/tutorial-install-python-bindings.dox

```diff
@@ -672,26 +672,50 @@ If you have an issue that looks like:
   /usr/include/c++/11/bits/move.h:196:5:   required by substitution of ‘template<class _Tp> std::_Require<std::__not_<std::__is_tuple_like<_Tp> >, std::is_move_constructible<_Tp>, std::is_move_assignable<_Tp> > std::swap(_Tp&, _Tp&) [with _Tp = vpImage<double>]’
-  /home/sfelton/software/visp-sfelton/modules/core/include/visp3/core/vpImage.h:341:15:   required from ‘class vpImage<double>’
-  /home/sfelton/software/visp-sfelton/modules/core/include/visp3/core/vpImage.h:369:17:   required from here
+  /home/user/software/visp-user/modules/core/include/visp3/core/vpImage.h:341:15:   required from ‘class vpImage<double>’
+  /home/user/software/visp-user/modules/core/include/visp3/core/vpImage.h:369:17:   required from here
   /usr/include/c++/11/type_traits:1010:52: error: static assertion failed: template argument must be a complete class or an unbounded array
   1010 |       **static_assert(std::__is_complete_or_unbounded(__type_identity<_Tp>{}),**
 \endverbatim
 
 You should delete the files in `modules/python/` of the **build** directory.
 
+
+If you have an error that looks like this:
+
+\verbatim
+/home/user/miniconda3/envs/visp-python11/include/pybind11/cast.h: In instantiation of ‘pybind11::arg_v::arg_v(pybind11::arg&&, T&&, const char*) [with T = vpColVector (&)(const vpColVector&, const vpColVector&)]’:
+/home/user/miniconda3/envs/visp-python11/include/pybind11/cast.h:1452:53:   required from ‘pybind11::arg_v::arg_v(const char*, T&&, const char*) [with T = vpColVector (&)(const vpColVector&, const vpColVector&)]’
+/home/user/visp_ws/visp_ukf/modules/python/bindings/src/core.cpp:20491:347:   required from here
+/home/user/miniconda3/envs/visp-python11/include/pybind11/cast.h:1432:82: error: call of overloaded ‘cast(vpColVector (&)(const vpColVector&, const vpColVector&), pybind11::return_value_policy, <brace-enclosed initializer list>)’ is ambiguous
+ 1432 |         : arg(base), value(reinterpret_steal<object>(detail::make_caster<T>::cast(
+      |                                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
+ 1433 |                          std::forward<T>(x), return_value_policy::automatic, {}))),
+      |                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In file included from /home/user/miniconda3/envs/visp-python11/include/pybind11/cast.h:15,
+                 from /home/user/miniconda3/envs/visp-python11/include/pybind11/attr.h:14,
+                 from /home/user/miniconda3/envs/visp-python11/include/pybind11/detail/class.h:12,
+                 from /home/user/miniconda3/envs/visp-python11/include/pybind11/pybind11.h:13,
+                 from /home/user/visp_ws/visp_ukf/modules/python/bindings/src/core.cpp:3:
+\endverbatim
```

Comment:
Instead of `/home/user/visp_ws/visp_ukf/` use `/home/user/visp_ws/visp/`

Reply:
Done in 407c96aa0b683ac4183b234ca0e5d6000ef4d22c